details.js

```diff
@@ -0,0 +1,150 @@
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    desktopClass: 'desktop',
+    reverseClass: 'reverse',
+  },
+];
+
+for (let i = 0; i < Articles.length; i += 1) {
+  const myArticles = document.getElementById('work_section').innerHTML;
+  workSection.innerHTML = `<article class="${Articles[i].articleClass}">
+                <div class="work_img">
+                    <img class="mobile" src="${Articles[i].mobileImg}" alt="work screenshot">
+                    <img class="desktop" src="${Articles[i].desktopImg}">
+                </div>
+                <div class="work_info">
+                    <h2>${Articles[i].heading}</h2>
+                    <div class="canopy">
+                        <div>
+                            <h5>CANOPY</h5>
+                        </div>
+                        <ul class="canopy_languages">
+                            <li>
+                                <img src="images/Counter.png" alt="counter image">
+                                <span>Backend dev</span>
+                            </li>
+                            <li>
+                                <img src="images/Counter.png" alt="counter image">
+                                <span>2015</span>
+                            </li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class= "see" type="button">See Project</button>
+                </div>
+
+            </article>
+             <!--pop up-->
+        <div class="desktopPopup">
+            <div class="desktopPopup_header">
+                <h2>Tonic</h2>
+                <button type='button' id= "closePopup" class="closePopup">x</button>
+            </div>
+            <div class="canopy ">
+                <div>
+                    <h5>CANOPY</h5>
+                </div>
+                <ul class="canopy_languages">
+                    <li>
+                        <img class="smallImg" src="images/Counter.png" alt="counter image ">
+                        <span>Backend dev</span>
+                    </li>
+                    <li>
+                        <img class="smallImg" src="images/Counter.png" alt="counter image ">
+                        <span>2015</span>
+                    </li>
+                </ul>
+            </div>
+            <div class="popupImg">
+                <img src="${Articles[i].desktopImg}" alt="popup image">r
+            </div>
+            <div class="desktopPopup_descriptions">
+                <div>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, sequi fuga veniam labore
```

Comment:
Great job on the mobile version! However, there's a small adjustment needed to match the design requirements:

1. Font size: Currently, the font size is set to 10px, but as per the design, it should be increased. Please follow the design and update the font styles, including line height, font size, etc., to match the design.

By making this adjustment, your mobile version will be more visually aligned with the design, creating a better user experience. Keep up the good work, and I'm sure your project will turn out amazing! 👏📱🚀